docs/src/userguide/glossary.rst

```diff
@@ -0,0 +1,194 @@
+        so that not all data is in RAM at once.
+
+        | **Related:** :term:`Lazy Data` **|** :term:`NumPy`
+        | **More information:** :doc:`real_and_lazy_data`
+        |
+
+   Fields File (FF) Format
+        A meteorological file format, the output of the Unified Model.
+
+        | **Related:**  :term:`GRIB Format`
+         **|** :term:`Post Processing (PP) Format` **|** :term:`NetCDF Format`
+        | **More information:** `Unified Model <https://www.metoffice.gov.uk/research/approach/modelling-systems/unified-model/index>`_
+        |
+
+   GRIB Format
+        A WMO-standard meteorological file format.
+
+        | **Related:** :term:`Fields File (FF) Format`
+         **|** :term:`Post Processing (PP) Format` **|** :term:`NetCDF Format`
+        | **More information:** `GRIB 1 User Guide <https://old.wmo.int/extranet/pages/prog/www/WMOCodes/Guides/GRIB/GRIB1-Contents.html>`_
+         **|** `GRIB 2 User Guide.pdf <https://old.wmo.int/extranet/pages/prog/www/WMOCodes/Guides/GRIB/GRIB2_062006.pdf>`_
+        |
+
+   Lazy Data
+        Data stored in hard drive, and then temporarily loaded into RAM in
+        batches when needed. Allows of less memory usage and faster performance,
+        thanks to parallel processing.
+
+        | **Related:** :term:`Dask` **|** :term:`Real Data`
+        | **More information:** :doc:`real_and_lazy_data`
+        |
+
+   Long Name
+        A name describing a :term:`phenomenon`, not limited to the
+        the same restraints as :term:`standard name`.
+
+        | **Related:** :term:`Standard Name` **|** :term:`Cubes`
+        | **More information:** :doc:`iris_cubes`
+        |
+
+   Matplotlib
+        A python package for plotting and projecting data in a wide variety
+        of formats.
+
+        | **Related:** :term:`CartoPy` **|** :term:`NumPy`
+        | **More information:** `Matplotlib <https://scitools.org.uk/cartopy/docs/latest/>`_
+        |
+
+   Metadata
+        The data which is used to describe phenomenon data e.g. longitude.
+
+        | **Related:** :term:`Phenomenon` **|** :term:`Cubes`
+        | **More information:** :doc:`../further_topics/metadata`
+        |
+
+   NetCDF Format
+        A meteorological file format; this is the data model
+        iris is based on. Follows `CF Conventions <http://cfconventions.org/>`_.
+
+        | **Related:** :term:`Fields File (FF) Format`
+         **|** :term:`GRIB Format` **|** :term:`Post Processing (PP) Format`
+        | **More information:** `NetCD-4 Python Git <https://github.com/Unidata/netcdf4-python>`_
+        |
+
+   NumPy
+        A mathematical Python library, predominantly based around
+        multi-dimensional arrays.
+
+        | **Related:** :term:`Dask`  **|** :term:`Cubes`
+         **|** :term:`Xarray`
+        | **More information:** `NumPy.org <https://numpy.org/>`_
+        |
+
+   Phenomenon
+        The primary data which is measured, usually within a cube, e.g.
+        air temperature.
```

Comment:
```suggestion
        The physical meaning of data array values, e.g. `cube.data` or `coordinate.points`. For example : air temperature.
```

Reply:
This took me a couple reads to entirely understand; trying to maintain the amount of information but perhaps more beginner friendly. 
Might leave for the time being; I'm struggling to find that middle ground; maybe something to bring up after merge.

Reply:
TBH I think it's ok as it was.  On reflection, I think my adding mention of coordinates was a bad call !